etc/env/example.env

```diff
@@ -181,9 +180,9 @@ KEYCLOAK_HOST=0.0.0.0
 KEYCLOAK_PUBLIC_PROTO=http
 KEYCLOAK_PRIVATE_PROTO=http
 KEYCLOAK_ENABLE_PROXY=false
-KEYCLOAK_PUBLIC_URL=${KEYCLOAK_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_PORT}
+KEYCLOAK_PUBLIC_URL=${KEYCLOAK_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}
 KEYCLOAK_PUBLIC_URL_PROD=${KEYCLOAK_PUBLIC_PROTO}://${CANDIG_AUTH_DOMAIN}
-KEYCLOAK_PRIVATE_URL=${KEYCLOAK_PRIVATE_PROTO}://${CANDIG_AUTH_DOMAIN}:${KEYCLOAK_CONTAINER_PORT}
+KEYCLOAK_PRIVATE_URL=${KEYCLOAK_PRIVATE_PROTO}://keycloak:${KEYCLOAK_CONTAINER_PORT}
```

Comment:
We should try this value out on other servers before making this switch for sure, but I think this is probably ok?